core/chains/evm/gas/suggested_price_estimator_test.go

```diff
@@ -25,8 +25,9 @@ func TestSuggestedPriceEstimator(t *testing.T) {
 
 	calldata := []byte{0x00, 0x00, 0x01, 0x02, 0x03}
 	const gasLimit uint64 = 80000
+	const limitMultiplier = 1.5
```

Comment:
Since the multiplier logic has been moved out of individual estimators, think we can clean these tests changes

Reply:
Just cleaned all the unnecessary limit multiplier logic from all the tests, except for the block history estimator tests, since the [block history's implementation](https://github.com/smartcontractkit/chainlink/blob/ffc2cf25f34bc91ff04feb066e748ac4b0da4360/core/chains/evm/gas/block_history_estimator.go#L471) of `BumpDynamicFee` uses the evm gas estimator's `BumpDynamicFeeOnly` which [applies the limit multiplier ](https://github.com/smartcontractkit/chainlink/blob/773d9b01ce137fc1ef59989d106341fa2333b3b2/core/chains/evm/gas/models.go#L418)